src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -420,7 +431,8 @@ def post_request(
             redir,
             timeout)
 
-        if isinstance(data, str):
+        logger.debug(type(data))
+        if hasattr(data, 'decode'):
```

Comment:
In python3 it will already be correct:
```
In [21]: a = ''

In [22]: hasattr(a, 'decode')
Out[22]: False

In [23]: a = b''

In [24]: hasattr(a, 'decode')
Out[24]: True
```